### DISCUSSION TITLE
PaddleOCR expects images in  RGB or BGR format  ?

### DISCUSSION THREAD

**Discussion by a-turky** [PaddlePaddle/PaddleOCR#13996] [Q&A] [answered]
Hello, 

I am using open cv to extract the image (frame) form the video. I would like to use after that the paddleOCR to extract the text within this frame.

Therefore, I would like to be very sure about if the PaddleOCR expects the images in  RGB or BGR format  ?

I would appreciate your help and thanks in advance for your cooperation and support

**Comment by a-turky** [comment]
Could any body please help me here and write short answer for this question?
Thanks in advance for your help,

**Comment by GreatV** [comment, marked as answer]
https://github.com/PaddlePaddle/PaddleOCR/blob/7bac0513b0cb0522e8b623ddb5b6f2f66dc28ef5/configs/det/ch_PP-OCRv3/ch_PP-OCRv3_det_student.yml#L136-L137

**Reply by a-turky** [reply]
Thank a lot :)
